src/ArrayInterface.jl

```diff
@@ -476,7 +476,7 @@ cheaply.
 function cholesky_instance(A::Matrix{T}, pivot = LinearAlgebra.RowMaximum()) where {T}  
     return cholesky(similar(A, 0, 0), pivot, check = false)
```

Comment:
I don't know about the dense case, but it normally isn't necessary to pivot cholesky factorizations.